docs/src/configuration/app/hooks/hooks-comparison.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Comparison of hooks
+description: What hooks are available in the build and deploy process and how to choose which to use.
+---
+
+The following table presents the main differences among the three available hooks:
+
+| Hook name     | Failures stop build | Variables available | Services available | Run on `worker` instances | Writable directories | Blocks requests | Runs on all redeploys\* |
+| ------------- | ------------------- |------------------- | ------------------ | ------------------------- | -------------------- | --------------- | --------------- |
+| `build`       | Yes                 | Build variables     | No                 | Yes                       | `$PLATFORM_APP_DIR`, `$PLATFORM_CACHE_DIR`, and `/tmp` | No  | No  |
```

Comment:
At first, I wanted to do this. But it would also mean changing the header. And making it slightly more complex.

And given the sticky table of contents on the side, I don't think the need for it is so strong. I could be wrong, but since those are the only three headings I think in this case it works without it.

If we had clear links on inline code, I think I'd add it, but we don't yet.

Reply:
That is true, the idea was to make navigation a little bit more comfortable. But since the page is not that big, the current behavior should already be enough as is.